pipeline/relation_embedder/path_concatenator/src/test/scala/weco/pipeline/path_concatenator/PathConcatenatorWorkerServiceTest.scala

```diff
@@ -0,0 +1,160 @@
+package weco.pipeline.path_concatenator
+
+import com.sksamuel.elastic4s.Index
+import org.scalatest.Assertion
+import org.scalatest.concurrent.{Eventually, IntegrationPatience}
+import org.scalatest.funspec.AnyFunSpec
+import weco.akka.fixtures.Akka
+import weco.catalogue.internal_model.index.IndexFixtures
+import weco.catalogue.internal_model.work.WorkState.Merged
+import weco.catalogue.internal_model.work.generators.WorkGenerators
+import weco.catalogue.internal_model.work.{CollectionPath, Work}
+import weco.elasticsearch.test.fixtures.ElasticsearchFixtures
+import weco.fixtures.TestWith
+import weco.messaging.fixtures.SQS.QueuePair
+import weco.messaging.memory.MemoryMessageSender
+import weco.messaging.sns.NotificationMessage
+import weco.pipeline_storage.fixtures.PipelineStorageStreamFixtures
+import weco.pipeline_storage.memory.MemoryIndexer
+import weco.catalogue.internal_model.Implicits._
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+/**
+  * Tests covering the Path Concatenator Worker Service, which
+  * responds to SQS messages to change records in a given Elasticsearch database
+  *
+  * These tests require running instances of
+  * - ElasticSearch
+  * - Localstack
+  *   - docker run --env SERVICES=sqs -p4566:4566 localstack/localstack:0.12.20
+  */
```

Comment:
I added it because I have found this out after the fact when running the tests in other services, by seeing suites go all red in IntelliJ.  I thought that a reminder at the top of the suite might be handy.